Tests/ComposableArchitectureTests/StoreTests.swift

```diff
@@ -32,14 +32,14 @@ final class StoreTests: XCTestCase {
 
       enum Action { case start, end }
 
-      let reducer = Reduce<Void, Action> { _, action in
+      let reducer = Reduce<Void, Action>({ _, action in
```

Comment:
All of these changes in StoreTests.swift are just to help swift out with the ambiguity errors. This only happens because of the `@testable import`. Without that it is not ambiguous.